docs/mine-hnt/validators/requirements.mdx

```diff
@@ -13,13 +13,23 @@ Based on early tests, validator node requirements include:
   - Planning on running from docker: Any linux based system capable of running docker
   - Planning on running from source: Ubuntu 20.04 LTS
 - CPU: 
-  - Intel, AMD or ARM based processor 3.0 GHz or above with 2 cores or equivalent
+  - Intel, AMD or ARM based processor 3.0 GHz or above with 4 cores or equivalent. 
   - In testnet, scores of [GeekBench 5](https://www.geekbench.com/) Multi-Core of greater than 2,200 have performed well
   - For Intel and AMD systems, [AVX2 or AVX support](https://en.wikipedia.org/wiki/Advanced_Vector_Extensions) is required. Verify your CPU supports these extensions by checking `grep avx /proc/cpuinfo` and verifying that AVX support is included in the output.
   - These recommendations may change for mainnet as the size of consensus groups grow and hotspot transactions are added
 - Memory: 8 GB or above
-- Storage: 256 GB or more for mainnet.  64GB is sufficient for testnet.
-- If using AWS, a t2.large or t2.xlarge EC2 instance or equivalent is recommended. If using another cloud provider, like GCP, Azure, or Digital Ocean, an instance with similar capabilities should be the target.
+- Storage: 256 GB or more for mainnet.  64GB is sufficient for testnet. This should not be on the OS disk and remoting your `/home/*` directory on a seperate disk is recommended.
```

Comment:
A few things:
* I don't know what "remoting" is. I think you mean: "putting your home directory on a separate disk is recommended"
* You don't have to put your whole home dir on another disk, in fact this can make troubleshooting more difficult if things go wrong (try logging in if the 2nd disk can't mount). 

How about this:
>It is recommended that you keep your validator data on a separate disk.

Reply:
Reworded